src/Handlers/MainContainer.cpp

```diff
@@ -19,7 +22,8 @@ int MainContainer::Init () {
 			c.append_attribute("cross").set_value("GPU");
 		#endif
 		solver->configfile.save_file(filename);
```

Comment:
I see no problem in processing the altered XML. Maybe even in future we will want to add things to the XML on-the-go (by python for instance)